src/advanced-concepts/custom_signature_validation.md

```diff
@@ -0,0 +1,17 @@
+# Custom Signature Validation Scheme
+
+Account Abstraction on Starknet supports various signature schemes. This means that signature schemes on Starknet are not limited to just one, any standard signature scheme can be validated, for example Starknet signature, Secp256k1, Secp256r1, Eip191 et al are some of the custom signatures that can be validated on Starknet currently. 
```

Comment:
> Account Abstraction on Starknet supports various signature schemes.

It's more that because Starknet have Account Abstraction, anyone can implement it's own custom signature scheme.